Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM python:3 as base
+RUN mkdir /opt/todoapp
+RUN chmod 775 /opt/todoapp
+WORKDIR /opt/todoapp
+COPY . .
+#COPY todo_app/* todo_app/
+RUN pip install poetry 
+RUN pip install --no-cache-dir pyproject.toml
+RUN pip install gunicorn flask 
+#RUN pip install --use-pep517 dotenv
+ENV WEBAPP_PORT=8000
+EXPOSE ${WEBAPP_PORT}
```

Comment:
Given that this is only true for the production image (since dev runs on port 5000 by default) it would be nice to shift this into that stage